phac-nml/irida-next
# add sapporo docs

## What does this PR do and why?
_Describe in detail what your merge request does and why._
Reliant on #507

Removes the documentation for ga4gh starter kit
Adds documentation for sapporo ga4gh wes implementation

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Start the docs locally

```bash
cd docs-site
pnpm build
npm run serve
```

Check that the new docs page looks correct

Then follow the instructions as written and verify that everything works and runs correctly. If you need to deviate from the instructions please indicate in the PR review for corrected instructions.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
